lib/rubocop/cop/rspec/missing_example_group_argument.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks that the first argument to an example group is not empty.
+      #
+      # You can configure a list of aliases of `example_group` that should be
+      # checked.
+      #
+      # @example
+      #   # bad
+      #   describe do
+      #   end
+      #
+      #   RSpec.describe do
+      #   end
+      #
+      #   # good
+      #   describe TestedClass do
+      #   end
+      #
+      #   describe "A feature example" do
+      #   end
+      class MissingExampleGroupArgument < Cop
+        MSG = 'The first argument to `%<method>s` should not be empty.'.freeze
+
+        def on_send(node)
+          return unless method_names.include?(node.method_name)
+          return unless node.first_argument.nil?
```

Comment:
You can use just `return if node.arguments?`

Reply:
Good point. Thanks.